lib/tasks/inaturalist.rake

```diff
@@ -47,7 +47,12 @@ namespace :inaturalist do
   end
 
   desc "Delete expired updates"
-  task delete_expired_updates: :environment do
+  task :delete_expired_updates, [:log_task_name] => :environment do | _, args |
+    log_task_name = args[:log_task_name]
+    if log_task_name
+      task_logger = TaskLogger.new( log_task_name, nil, "cleanup", "rails" )
```

Comment:
With `rails` being the default value for subtype it's not necessary to include this last parameter anymore